terraform/modules/ops/services.tf

```diff
@@ -11,6 +11,12 @@ locals {
   beta_services = var.enable_apis ? [
     "artifactregistry.googleapis.com"
   ] : []
+  # Cloud build service account roles
```

Comment:
This works, but to keep this file just for services, add a local block with this value at the top of `main.tf` instead.